docs/releasenotes/release-1.120.md

```diff
@@ -25,6 +25,8 @@ output fields from GCP APIs are in `status.observedState.*`
   * Added `spec.severity` field.
 
 * `MonitoringDashboard`
+  * Added `alertChart` widgets.
+  * Added `collapsibleGroup` widgets.
```

Comment:
I don't see `collapsibleGroup` in this PR (guess that's due to some diffbase). Hope this won't cause conflict.

Reply:
Right - it's because we've already merged it, I just forgot to update the release notes!